src/firefly/js/api/ApiUtil.js

```diff
@@ -35,8 +35,8 @@ export const isDebug = () => get(window, 'firefly.debug', false);
  * @param {String|Error} msg any number of messages
  */
 export function debug(...msg) {
-    if (!isEmpty(msg) && isDebug()) {
-        logError(...msg);
+    if (isDebug() && !isEmpty(msg)) {
+        logErrorWithPrefix('Firefly:', ...msg);
```

Comment:
you can pass a single string. That is what I am doing.   However, you can also pass multiple parameters. Such as
```
debug('message 1', new Error('this is the error', {a: 'error obj', b: 'more' });
```
When it comes out it will start with 'Firefly: '